CMakeLists.txt

```diff
@@ -35,10 +35,14 @@ endif()
 
 include(FetchContent)
 
+# Set up Elastix build parameters for populate step
+option(ELASTIX_BUILD_TESTING OFF)
+option(ELASTIX_BUILD_SHARED OFF)
```

Comment:
I like the proposed `ELASTIX_BUILD_TESTING` option, but I wonder, why do you propose to add `ELASTIX_BUILD_SHARED`? While I think the default (`OFF`) is fine, SuperElastix/elastix just does not support BUILD_SHARED_LIBS (`ON`), as far as I know... (See also https://github.com/SuperElastix/elastix/pull/283 "COMP: No longer add BUILD_SHARED_LIBS option").

Reply:
Thanks for the catch @N-Dekker, I wrongly assumed Elastix supported shared libraries. Not a problem, will remove.

Reply:
Updated.